data/eventStorage.json

```diff
@@ -0,0 +1,141 @@
+{
+  "events" : [ {
+    "eventName" : "fixing computers",
+    "roles" : [ {
+      "roleName" : "mechanic",
+      "currentQuantity" : "0",
+      "requiredQuantity" : "1"
+    }, {
+      "roleName" : "engineer",
+      "currentQuantity" : "0",
+      "requiredQuantity" : "4"
+    } ],
+    "startDate" : "10/1/2023 1600",
+    "endDate" : "14/5/2023 1500",
+    "location" : "home",
+    "description" : "fix computers",
+    "materials" : [ {
+      "material" : "computers",
+      "currentQuantity" : "0",
+      "requiredQuantity" : "50"
+    } ],
+    "budget" : "120.00",
+    "assignedVolunteers" : [ ],
+    "maxVolunteerSize" : "9223372036854775807"
+  }, {
+    "eventName" : "picking litter",
+    "roles" : [ {
+      "roleName" : "litterer",
+      "currentQuantity" : "0",
+      "requiredQuantity" : "4"
+    }, {
+      "roleName" : "picker",
+      "currentQuantity" : "0",
+      "requiredQuantity" : "10"
+    } ],
+    "startDate" : "10/3/2023 1500",
+    "endDate" : "20/3/2023 1900",
+    "location" : "queenstown",
+    "description" : "picking litter at queenstown",
+    "materials" : [ {
+      "material" : "tongs",
+      "currentQuantity" : "0",
+      "requiredQuantity" : "10"
+    } ],
+    "budget" : "40.00",
+    "assignedVolunteers" : [ "Ben" ],
+    "maxVolunteerSize" : "9223372036854775807"
+  }, {
+    "eventName" : "food donation",
+    "roles" : [ {
+      "roleName" : "cook",
+      "currentQuantity" : "0",
+      "requiredQuantity" : "50"
+    }, {
+      "roleName" : "donator",
+      "currentQuantity" : "0",
+      "requiredQuantity" : "50"
+    } ],
+    "startDate" : "13/4/2023 1500",
+    "endDate" : "15/4/2023 1900",
+    "location" : "sengkang",
+    "description" : "donating food",
+    "materials" : [ {
+      "material" : "vegetables",
+      "currentQuantity" : "0",
+      "requiredQuantity" : "50"
+    } ],
+    "budget" : "500.00",
+    "assignedVolunteers" : [ ],
+    "maxVolunteerSize" : "9223372036854775807"
+  }, {
+    "eventName" : "baking cookies",
+    "roles" : [ {
+      "roleName" : "baker",
+      "currentQuantity" : "0",
+      "requiredQuantity" : "30"
+    }, {
+      "roleName" : "dough",
```

Comment:
hm i think we shld be putting actual roles tho i j scared the testers guailan

Reply:
aiya ok